tfhe/src/high_level_api/array/gpu/integers.rs

```diff
@@ -0,0 +1,648 @@
+//! This module contains the implementations of the FheUint array and FheInt array backend
+//! where the values and computations are always done on CPU
```

Comment:
```suggestion
where the values and computations are always done on CPU
```

Reply:
If I remove `//!` the line won't be a comment anymore, but I have to replace CPU with GPU, is it what you had in mind?

Reply:
haha no my suggestion is wrong, I meant to change CPU to GPU 😓